modules/about-workbench-images.adoc

```diff
@@ -45,7 +45,7 @@ endif::[]
 | Use the TrustyAI notebook image to leverage your data science work with model explainability, tracing, and accountability, and runtime monitoring.
 
 | HabanaAI
-| The HabanaAI notebook image optimizes high-performance deep learning (DL) with Habana Gaudi devices. Habana Gaudi devices accelerate DL training workloads and maximize training throughput and efficiency.
+| The HabanaAI notebook image optimizes high-performance deep learning (DL) with Intel Gaudi AI Accelerators. Intel Gaudi AI Accelerators accelerate DL training workloads and maximize training throughput and efficiency.
```

Comment:
I assume that the HabanaAI notebook image name still contains the word "Habana", and has not changed.

Reply:
Yes - the image and the HabanaAI Operator naming aren't changing currently.